config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -173,6 +173,11 @@ config NFC_THREAD_CALLBACK
 config CHIP_OTA_REQUESTOR
     default y
 
+if THREAD_WIFI_SWITCHING
```

Comment:
This is defined just in the lock sample, right? If so, it may be better to modify `DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` there?

Reply:
Ok, I will move it to the lock sample.